### ISSUE TITLE
This plugin no longer works in Ember 1.13.1

### ISSUE THREAD

**Issue by jackmatt2** [pzuraq/ember-inputmask#9]
```
Uncaught TypeError: Cannot read property 'inputmask' of undefined(anonymous function) @ input-mask.js:86applyStr @ ember.debug.js:21939sendEvent @ ember.debug.js:15479notifyObservers @ ember.debug.js:18992propertyDidChange @ ember.debug.js:18788exports.default._emberMetalMixin.Mixin.create.propertyDidChange @ ember.debug.js:34847exports.default._emberMetalMixin.Mixin.create.notifyPropertyChange @ ember.debug.js:34861(anonymous function) @ ember.debug.js:41903applyStr @ ember.debug.js:21937sendEvent @ ember.debug.js:15479exports.default._emberMetalMixin.Mixin.create.trigger @ ember.debug.js:34000superFunction @ ember.debug.js:17024_emberRuntimeSystemObject.default.extend.trigger @ ember.debug.js:45999superWrapper @ ember.debug.js:21545Renderer.componentUpdateAttrs @ ember.debug.js:11469(anonymous function) @ ember.debug.js:9269instrument @ ember.debug.js:10000ComponentNodeManager.rerender @ ember.debug.js:9263componentHook @ ember.debug.js:7607handleRedirect @ ember.debug.js:49601inline @ ember.debug.js:49755AlwaysDirtyVisitor.inline @ ember.debug.js:48836(anonymous function) @ ember.debug.js:48920dirtyCheck @ ember.debug.js:48968exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440block @ ember.debug.js:51005(anonymous function) @ ember.debug.js:9285instrument @ ember.debug.js:10000ComponentNodeManager.rerender @ ember.debug.js:9263componentHook @ ember.debug.js:7607handleRedirect @ ember.debug.js:49601inline @ ember.debug.js:49755AlwaysDirtyVisitor.inline @ ember.debug.js:48836(anonymous function) @ ember.debug.js:48920dirtyCheck @ ember.debug.js:48968exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786validateChildMorphs @ ember.debug.js:50782dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786validateChildMorphs @ ember.debug.js:50782dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912RenderResult.populateNodes @ ember.debug.js:50466RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.inline @ ember.debug.js:48919RenderResult.populateNodes @ ember.debug.js:50468RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.content @ ember.debug.js:48926RenderResult.populateNodes @ ember.debug.js:50470RenderResult.revalidateWith @ ember.debug.js:50440validateChildMorphs @ ember.debug.js:50786dirtyCheck @ ember.debug.js:48973exports.default.block @ ember.debug.js:48912
```

**Comment by neojp** [comment]
This has been addressed on this PR https://github.com/pzuraq/ember-inputmask/pull/8

**Comment by jackmatt2** [comment]
Has this been updated on npm because I tried it on Friday and it still didn't work.

**Comment by pzuraq** [comment]
@jackmatt2 published it as 0.2.0 just now
